docs/docs/testing-react-components.md

```diff
@@ -29,8 +29,16 @@ import "react-testing-library/cleanup-after-each"
 
 This file gets run automatically by Jest before every test and therefore you don't need to add the imports to every single test file.
 
-Lastly you need to tell Jest where to find this file. Open your `package.json` and add this entry to your `"jest"` section:
+Lastly you need to tell Jest where to find this file. Assuming you followed the [Unit testing](/docs/unit-testing), open your `jest.config.js` and add this entry to the bottom after 'setupFiles':
 
+```js:title=jest.config.js
+module.exports = {
+  ...
+  'setupTestFrameworkScriptFile': '<rootDir>/setup-test-env.js'
+}
+```
+
+*If you don't have a `jest.config.js` file, you need to instead add this to your `package.json` like so:*
```

Comment:
In my opinion you can also cut the old part. It's not relevant anymore and it could confuse people

Reply:
Agreed! We should always show the simplest/clearest path. Showing multiple just leads to confusion, e.g. why would I use a file vs. package.json?

Reply:
Good point! Let me cut that.